_posts/2018-05-24-kotlin-project-setup.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: Kotlin Gradle projects and gRPC
+published: false
+permalink: blog/kotlin-gradle-projects
+author: Spencer Fang
+author-link: https://github.com/zpencer
+company: Google
+company-link: https://www.google.com
+---
+
+# Kotlin Gradle projects and gRPC
+
+Did you know that gRPC Java now has out of box support for Kotlin projects built with Gradle? Kotlin is a modern, statically typed language developed by JetBrains that targets the JVM and Android. It is generally easy for Kotlin programs to interoperate with existing Java libraries. We have added support to the [protobuf-gradle-plugin](https://github.com/google/protobuf-gradle-plugin/releases) so that the generated Java libraries are automatically picked up by Kotlin. You can now add the protobuf-gradle-plugin to your Kotlin project, and use gRPC just like you would with a typical Java project.
+
+Below are project configuration examples for a JVM application and an Android application using Kotlin.
```

Comment:
The follow examples show you how to configure a project for a JVM application and an Android application using Kotlin.

Reply:
Done